invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentPendingLoans/DocumentPendingLoans.js

```diff
@@ -14,7 +15,7 @@ export default class DocumentPendingLoans extends Component {
     super(props);
     this.fetchPendingLoans = props.fetchPendingLoans;
     this.showDetailsUrl = viewLoanDetailsUrl;
-    this.showAllUrl = showAllLoansUrl;
+    this.showAllUrl = loanSearchQueryUrl;
```

Comment:
I wouldn't change names in re-assignments, or I would keep something descriptive, `this.showLoanList` or something. Also showAll is going to appear in a lot of places and won't ring a bell for old guys like me.